sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0 (2021-11-09)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Adds stable features and bug fixes from the preview releases.
```

Comment:
shall we list all the breaking changes here?

Reply:
I thought about it. We can do that, but listing everything exhaustively would be a very long list, esp. with all the renames.

Reply:
Made some minor updates